src/plugins/discover/public/application/discover_router.tsx

```diff
@@ -43,6 +44,9 @@ export const discoverRouter = (services: DiscoverServices, history: History, isD
           <Route path="/view/:id">
             <DiscoverMainRoute isDev={isDev} />
           </Route>
+          <Route path="/log-explorer">
+            <DiscoverLogExplorerRoute isDev={isDev} />
```

Comment:
FYI: Discover uses hash based routing.

The saved search ID lands as a query parameter, not part of the path.